homeassistant/components/velbus/manifest.json

```diff
@@ -5,5 +5,24 @@
   "requirements": ["velbus-aio==2021.11.7"],
   "config_flow": true,
   "codeowners": ["@Cereal2nd", "@brefra"],
-  "iot_class": "local_push"
+  "dependencies": ["usb"],
+  "iot_class": "local_push",
+  "usb": [
+    {
```

Comment:
Looks like these are limited to just the one vendor so this should be descriptive enough
http://www.linux-usb.org/usb.ids

Reply:
No, the same vendor has other usb ids, so we need the vid and pid

Reply:
I think we are in violent agreement 

My comment was referencing https://developers.home-assistant.io/docs/creating_integration_manifest/#usb noting that description probably isn't needed here.